pkg/apiserver/rest/usecase/cluster.go

```diff
@@ -135,15 +138,26 @@ func (c *clusterUsecaseImpl) preAddLocalCluster(ctx context.Context) error {
 				}
 				return err
 			}
+			return nil
 		}
 		return err
 	}
+	if localCluster.CreateTime.Before(model.LocalClusterCreatedTime) {
+		localCluster.CreateTime = model.LocalClusterCreatedTime
+		if err = c.ds.Put(ctx, localCluster); err != nil {
+			return err
+		}
+	}
 	return nil
 }
 
 func (c *clusterUsecaseImpl) ListKubeClusters(ctx context.Context, query string, page int, pageSize int) (*apis.ListClusterResponse, error) {
```

Comment:
The query parameter does not take effect.

Reply:
You mean query parameter in ListKubeClusters? If so, yes. Currently no. We do not have any fuzzy query mechanism supported.

Reply:
My suggestion is to remove paging and support search filtering.

Reply:
Why remove paging? Search filtering is necessary and I will implement it in another PR. But is it necessary to remove paging?

Reply:
Search filtering is added now in this PR.